basic_benchmark.rb

```diff
@@ -255,7 +255,7 @@
 
 YJIT_GIT_URL = BENCH_DATA["cruby_repo"] || "https://github.com/ruby/ruby"
 YJIT_GIT_BRANCH = BENCH_DATA["cruby_sha"] || "master"
-YJIT_PREV_REF = "v3_3_4"
+YJIT_PREV_REF = "v3_3_6"
```

Comment:
Small detail but might be good to comment on what this constant is. I generally wish that this codebase was more commented.

Reply:
No worries, just small comments in passing. I think you're doing a great job at incrementally improving this codebase bit by bit 👍

Reply:
Thank you!